lib/Doctrine/DBAL/Migrations/SqlFileWriter.php

```diff
@@ -77,12 +77,12 @@ public function write(array $queriesByVersion, $direction)
 
     private function buildMigrationFile(array $queriesByVersion, $direction)
     {
-        $string = sprintf("# Doctrine Migration File Generated on %s\n", date('Y-m-d H:i:s'));
+        $string = sprintf("# Doctrine Migration File Generated on %s" . PHP_EOL, date('Y-m-d H:i:s'));
```

Comment:
I vote -1 for forcing PHP_EOL in generated files. In any project involving multiple OS, the standard is generally to use LF for files, not the line endings of the OS on which the file was created